vendor/bitcoin_support/src/network.rs

```diff
@@ -71,7 +71,7 @@ mod test {
         assert_eq!(testnet, "test");
     }
 
-    fn assert_display<T: Display>(t: T) {}
+    fn assert_display<T: Display>(_t: T) {}
```

Comment:
:mega: I guess you shouldn't fix this unless you want to do #1139 too.

Reply:
It sounds like a nitpick but the ticket also includes to investigate why clippy doesn't fail this so it should probably not be done as part of this :)